### PR TITLE
 add wifi_power for insight device

### PR DESCRIPTION
**Pull request by minscof**
## Description:
see issue #154 

**Related issue (if applicable):** fixes #<pywemo issue number goes here>

## Checklist:
  - [ ] The code change is tested and works locally.
  - [x] There is no commented out code in this PR.

### CODE REVIEW

File: pywemo/ouimeaux_device/insight.py [hunk 1]
```diff
@@ -80,6 +81,11 @@ def today_kwh(self):
     def current_power(self):
         """Return the current power usage in mW."""
         return self.insight_params['currentpower']
+
```
Comment (sqldiablo) [inline pywemo/ouimeaux_device/insight.py]: It looks like there's whitespace on this line, and that's why the CI tests failed. Can you please remove the whitespace?

**Review by sqldiablo — CHANGES_REQUESTED** [review]

File: pywemo/ouimeaux_device/insight.py (hunk 1 above)
Comment (minscof) [reply pywemo/ouimeaux_device/insight.py]: Ok i remove the whitespace

File: pywemo/ouimeaux_device/insight.py [hunk 2]
```diff
@@ -80,6 +81,11 @@ def today_kwh(self):
...
```
Comment (sqldiablo) [inline pywemo/ouimeaux_device/insight.py]: It looks like there are still 3 blank spaces on line 84 of this file. Can you please make sure that line 84 is just a blank line?

**Review by sqldiablo — CHANGES_REQUESTED** [review]

**Comment by sqldiablo** [comment]
Thanks for taking care of the whitespace issue, @minscof . Linting is now passing. I'm having some trouble with my testing setup, but I will merge this in once I've been able to test it successfully.

**Comment by sqldiablo** [comment]
Tested successfully (although my Insight just returns a 0 for wifi_power). I'll go ahead and merge this in though. Thanks for your contribution.

**Comment by sqldiablo** [comment]
Published to Pypi as pywemo version 0.4.44

**Pull request merged**